source/Calamari.Tests/DeployAzureCloudServiceCommandFixture.cs

```diff
@@ -145,7 +156,7 @@ public async Task Deploy_Package_To_Stage_And_Swap()
                                                          context.Variables.Add(SpecialVariables.Action.Azure.UseCurrentInstanceCount, bool.FalseString);
                                                          context.Variables.Add(SpecialVariables.Action.Azure.DeploymentLabel, "v1.0.0");
 
-                                                         context.WithPackage(pathToPackage, "Octopus.Sample.AzureCloudService", "5.8.2");
+                                                         context.WithPackage(pathToPackage, "Octopus.Sample.AzureCloudService", "6.0.0");
                                                      })
                                         .Execute();
 
```

Comment:
Should assert prod exists too, is it worth asserting the status here?

Reply:
Done